packages/palette/src/elements/Dialog/Dialog.story.tsx

```diff
@@ -0,0 +1,59 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Dialog } from "./Dialog"
+
+storiesOf("Components/Dialog", module)
+  .add(
+    "Minimal",
+    () => {
+      return (
+        <Dialog
+          show
+          title="Here is a dialog which is modal"
+          primaryCta={{
+            action: () => ({}),
+            text: "Continue",
+          }}
+        />
+      )
+    },
+    { chromatic: { delay: 500, diffThreshold: 0.2 } }
+  )
+  .add(
+    "With detail",
+    () => {
+      return (
+        <Dialog
+          show
+          title="Information"
+          detail="This extra informaton is important."
+          primaryCta={{
+            action: () => ({}),
```

Comment:
Same here -- this shouldn't be required so as to keep the API requirements slender, and instead the ` undefined` action handled inside the implementation.

Reply:
I think the idea here is that at least one button should be required for a `Dialog`. We can default the text of it to something generic like `ok` and action is to just close the `dialog` but that is one of the distinctions of a `Dialog` if I understand it correctly. `Dialog` is something that blocks the flow with at least one action that should be actively acknowledged (clicked) to move with the flow.

Reply:
Yeah! A dialog presents the user critical information that require their attention/make a decision–purposely interruptive of their flow.